src/Sylius/Behat/Context/Api/EmailContext.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Sylius\Behat\Context\Api;
+
+use Behat\Behat\Context\Context;
+use Sylius\Component\Core\Test\Services\EmailCheckerInterface;
+use Symfony\Contracts\Translation\TranslatorInterface;
+use Webmozart\Assert\Assert;
+
+class EmailContext implements Context
```

Comment:
```suggestion
final class EmailContext implements Context
```

Reply:
I will fix/change it with refactor of EmailContext ( see comment https://github.com/Sylius/Sylius/pull/12360#discussion_r579044089 )